egui/src/data/output.rs

```diff
@@ -317,9 +351,24 @@ impl WidgetInfo {
     }
 
     #[allow(clippy::needless_pass_by_value)]
-    pub fn text_edit(edit_text: impl ToString) -> Self {
+    pub fn text_edit(text_value: impl ToString, prev_text_value: impl ToString) -> Self {
+        Self {
+            text_value: Some(text_value.to_string()),
+            prev_text_value: Some(prev_text_value.to_string()),
```

Comment:
"after, before" is a surprising order

```suggestion
    pub fn text_edit(prev_text_value: impl ToString, current_text_value: impl ToString) -> Self {
        Self {
            prev_text_value: Some(prev_text_value.to_string()),
            current_text_value: Some(current_text_value.to_string()),
```